Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml

```
@@ -255,3 +255,28 @@
   - type: ExplosionResistance
     damageCoefficient: 0.5
   - type: GroupExamine
+
+- type: entity
+  parent: ClothingOuterBaseLarge
+  id: ClothingOuterArmorBone
+  name: bone armor
+  description: Seats on you like a second skelet.
```

```suggestion
  description: Sits on you like a second skin.
```

or skeleton, no one uses skelet.